Library/Homebrew/global.rb

```diff
@@ -5,6 +5,7 @@
 require "extend/ARGV"
 require "PATH"
 require "extend/string"
+require "vendor/backports/string"
```

Comment:
I realized afterwards that `extend/string` is required by non-Homebrew code (like our rubocop extensions), and those don't have the right `$LOAD_PATH` to access the vendor directory. Going to move this back to `global`.